aredis_om/model/model.py

```diff
@@ -349,8 +367,9 @@ def __init__(
         self,
         expressions: Sequence[ExpressionOrNegated],
         model: Type["RedisModel"],
+        knn: Optional[KNNExpression] = None,
         offset: int = 0,
-        limit: int = DEFAULT_PAGE_SIZE,
+        limit: Optional[int] = None,
```

Comment:
PEP 484 suggests this should be:
```limit: Union[int, None] = None,```

Reply:
bump

Reply:
I thought PEP484 prohibits the implicit Optional, so it's prefered to do Union[T1, None]. Again, we don't currently have this throughout, so far.